### DISCUSSION TITLE
Real world example for DotNext.AspNetCore.Cluster

### DISCUSSION THREAD

**Discussion by harrison314** [dotnet/dotNext#113] [Ideas]
Hi,
i would like to know if you have some real world project example to use **DotNext.AspNetCore.Cluster** with http transport.

I looked at an example project on <https://github.com/dotnet/dotNext/tree/master/src/examples/RaftNode>, but the state represented by one `long` is not very explianing.  
Exmple on <https://medium.com/microsoftazure/geo-redundant-stateful-service-made-easy-with-asp-net-core-next-and-azure-virtual-machines-950843bb26fd> uses old versions of packages.

I'm looking for a examle with the REST API (and swagger), e.g. key-value distributed datbase with get key, insert and delete operation, like configuration store (litte Consul).

**Comment by sakno** [comment]
You asking for a complete distributed application, not for an example :smile: Example in repository demonstrates key aspects of Raft - leader election and replication. You can take a look at [Slik](https://github.com/Insvald/Slik) project. It provides distributed cache on top of Raft. Also, contribution is welcome. You can try to update an example in the article you mentioned.

P.S.: This is not a real issue, converting to Discussion.

**Comment by harrison314** [comment]
> You asking for a complete distributed application, not for an example

That is true: D

> You can take a look at [Slik](https://github.com/Insvald/Slik) project.

Thank you, that project can help me.
